src/test/java/com/ing/data/cassandra/jdbc/UtilsUnitTest.java

```diff
@@ -45,6 +45,13 @@
 
     static Stream<Arguments> buildUrlParsingTestCases() {
         return Stream.of(
+            Arguments.of("jdbc:cassandra://localhost:9042/astra?secureconnectbundle=/path/to/location/filename.extn",
+                new HashMap<String, String>() {{
+                    put(Utils.TAG_SERVER_NAME, "localhost");
+                    put(Utils.TAG_PORT_NUMBER, "9042");
+                    put(Utils.TAG_DATABASE_NAME, "astra");
+                    put(Utils.TAG_CLOUD_SECURE_CONNEC_BUNDLE, "/path/to/location/filename.extn");
+                }}),
```

Comment:
Maybe we could here also test the `username` and `password` arguments since they are not tested elsewhere in this unit tests class.
```suggestion
            Arguments.of("jdbc:cassandra://localhost:9042/astra?secureconnectbundle=/path/to/location/filename.extn&user=user1&password=password1",
                new HashMap<String, String>() {{
                    put(Utils.TAG_SERVER_NAME, "localhost");
                    put(Utils.TAG_PORT_NUMBER, "9042");
                    put(Utils.TAG_DATABASE_NAME, "astra");
                    put(Utils.TAG_CLOUD_SECURE_CONNECT_BUNDLE, "/path/to/location/filename.extn");
                    put(Utils.TAG_USER, "user1");
                    put(Utils.TAG_PASSWORD, "password1");
                }}),
```

Reply:
@maximevw I tried this, but it fails at other places for e.g. below:
```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   UtilsUnitTest.givenJdbcUrl_whenParseUrl_returnExpectedProperties:121->lambda$givenJdbcUrl_whenParseUrl_returnExpectedProperties$0:121 expected: <password1> but was: <null>
[INFO] 
[ERROR] Tests run: 176, Failures: 1, Errors: 0, Skipped: 0
```

and realized, we never captured `user` & `password` information as part of the parsing the input URL and have also added code snippets to address that along with my latest push.

Reply:
@msmygit You're right. Indeed, I didn't see these parameters were ignored. After all, adding this test was not a bad idea 😄  Great thanks for fixing this!